Content.Shared/Ninja/Components/NinjaComponent.cs

```diff
@@ -0,0 +1,38 @@
+using Content.Shared.Ninja.Systems;
+using Robust.Shared.GameStates;
+using Robust.Shared.Serialization;
+
+namespace Content.Shared.Ninja.Components;
+
+/// <summary>
+/// Component placed on a mob to make it a space ninja, able to use suit and glove powers.
+/// Contains ids of all ninja equipment.
+/// </summary>
+[RegisterComponent, NetworkedComponent, AutoGenerateComponentState]
+[Access(typeof(SharedNinjaSystem))]
+public sealed partial class NinjaComponent : Component
```

Comment:
rename to spaceninja

Reply:
everything??? pretty slim chance of regular ninja existing

Reply:
nah just the component and system